### DISCUSSION TITLE
Breakout rooms

### DISCUSSION THREAD

**Discussion by MurzNN** [element-hq/element-meta#86] [Ideas]
#### What would you like to do?
Will be good to have ability to create breakout rooms, like in Google Meet, Zoom and even recent Jitsi.

#### Why would you like to do it?
Very often we need the ability to temporary discuss some topic with only some people from all room members. 

Easy example can be a discussion about general birthday gift for one person in a chat from all other members in some team. So they need to discuss this in room with all members, excluding one or two (eg he and his wife). And after this discussion soon they must have a similar discussion about gift for next person, who have a birthday, but excluding him instead of previous one.

Now we have threads, but they are visible for all members, so not so suite for those needs.

Technically this can be a new separate Matrix room, into which all users are invited, and kicked after discussion is finished. But doing this in current interface is not so convenient.

#### How would you like to achieve it?
Implement some sort of interface for quick creating of breakout rooms, and quick forgetting about them after discussion is finished.

**Comment by kittykat** [comment]
I've transferred this issue to a discussion in element-meta as it is a cross-platform feature request :+1:
